pkg/client/didexchange/client.go

```diff
@@ -115,33 +116,40 @@ func New(ctx provider) (*Client, error) {
 func (c *Client) CreateInvitation(label string) (*Invitation, error) {
 	// TODO https://github.com/hyperledger/aries-framework-go/issues/623 'alias' should be passed as arg and persisted
 	//  with connection record
-	_, sigPubKey, err := c.legacyKMS.CreateKeySet()
+	kid, _, err := c.kms.Create(kms.ED25519Type)
 	if err != nil {
-		return nil, fmt.Errorf("failed CreateSigningKey: %w", err)
+		return nil, fmt.Errorf("createInvitation: failed to create SigningKey handle:%w", err)
 	}
 
+	sigPubKey, err := c.kms.ExportPubKeyBytes(kid)
+	if err != nil {
+		return nil, fmt.Errorf("createInvitation: failed to extract public SigningKey bytes from handle:%w", err)
+	}
+
+	sigPubKeyB58 := base58.Encode(sigPubKey)
+
 	// get the route configs
 	serviceEndpoint, routingKeys, err := mediator.GetRouterConfig(c.routeSvc, c.serviceEndpoint)
 	if err != nil {
-		return nil, fmt.Errorf("create invitation - fetch router config : %w", err)
+		return nil, fmt.Errorf("createInvitation: getRouterConfig: %w", err)
```

Comment:
Any specific reason to update the error messages ?

Reply:
`fetch router config ` was repeated by the wrapped error